components/canvas/utils/FormatNodeText.tsx

```diff
@@ -0,0 +1,34 @@
+import { Typography, TypographyProps } from "@equinor/eds-core-react";
+import styles from "../Node.module.scss";
+
+export function FormatNodeText({
+  children,
+  ...typographyProps
+}: TypographyProps) {
+  if (!children) return null;
+  const text = children.toString();
+  // Pattern matching for Markdown hyperlink urls
+  const pattern = /\[([^\]]+)]\(([^)]+)\)/g;
+  let result = [];
```

Comment:
Change this to const to avoid strict mode error

Reply:
fixed strict mode errors